featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, entityset, feature_set, time_last=None,
         # total number of features (including dependencies) to be calculate
         self.num_features = len(feature_set.features_by_name.values())
 
-    def run(self, instance_ids, progress_callback=None):
+    def run(self, instance_ids, update_progress_callback=None):
```

Comment:
do we need to update the name of the argument? I say this because I'm not sure the rename doesn't increase clarity and we don't have a naming conflict internal to this function